src/authenticationinapp/authenticationinapplistener.cpp

```diff
@@ -341,73 +341,73 @@ void AuthenticationInAppListener::finalizeSignInOrUp() {
             processRequestFailure(error, data);
           });
 
-  connect(request, &NetworkRequest::requestCompleted,
-          [this](const QByteArray& data) {
-            logger.debug() << "Oauth code creation completed" << data;
-
-            QJsonDocument json = QJsonDocument::fromJson(data);
-            if (json.isNull()) {
-              MozillaVPN::instance()->errorHandle(
-                  ErrorHandler::AuthenticationError);
-              return;
-            }
-
-            QJsonObject obj = json.object();
-            QJsonValue code = obj.value("code");
-            if (!code.isString()) {
-              logger.error() << "FxA Authz: code not found";
-              MozillaVPN::instance()->errorHandle(
-                  ErrorHandler::AuthenticationError);
-              return;
-            }
-            QJsonValue state = obj.value("state");
-            if (!state.isString()) {
-              logger.error() << "FxA Authz: state not found";
-              MozillaVPN::instance()->errorHandle(
-                  ErrorHandler::AuthenticationError);
-              return;
-            }
-            QJsonValue redirect = obj.value("redirect");
-            if (!redirect.isString()) {
-              logger.error() << "FxA Authz: redirect not found";
-              MozillaVPN::instance()->errorHandle(
-                  ErrorHandler::AuthenticationError);
-              return;
-            }
-
-            NetworkRequest* request =
-                NetworkRequest::createForGetUrl(this, redirect.toString(), 200);
-
-            connect(request, &NetworkRequest::requestFailed,
-                    [this](QNetworkReply::NetworkError error,
-                           const QByteArray& data) {
-                      logger.error()
-                          << "Failed to fetch the final redirect data" << error;
-                      processRequestFailure(error, data);
-                    });
+  connect(
```

Comment:
I can revert the commit that made this change if necessary.

It has nothing to do with this PR but the linter was failing.

Reply:
Let me fix this in a separate PR

Reply:
Done, rebase the main branch.